config/management/src/genesis.rs

```diff
@@ -62,11 +57,11 @@ impl Genesis {
         let mut common_config = self.backend.backend.clone();
         common_config
             .parameters
-            .insert("namespace".into(), COMMON_NS.into());
+            .insert("namespace".into(), crate::constants::COMMON_NS.into());
```

Comment:
Interesting.. I originally took this approach when I created the global constants crate, but I eventually decided to import the full path for two reasons:
(1) quoting from the rust book [1] : "On the other hand, when bringing in structs, enums, and other items with use, it’s idiomatic to specify the full path. " I took this to mean constants as well (although I've yet to find a concrete argument for/against this specific issue).
(2) it seemed ugly to me to have to import each constant using the full path, especially for files that use the same constant more than once..

It would be nice if there was some form of consensus on the issue (although I suspect not)? Leaving up to your discretion!

[1] https://doc.rust-lang.org/book/ch07-04-bringing-paths-into-scope-with-the-use-keyword.html#creating-idiomatic-use-paths

Reply:
I share your thoughts... my refinements is does importing it add more lines of code, if not don't import it.